Create rule S6361 (#427)
SonarSource/rspec: rules/S6361/description.adoc

``++android:permission++`` is used to set a single permission for both reading and writing data from a content provider.

> In regard to the Principle of Least Privilege, client applications that consume the content provider should only have the necessary privileges to complete their tasks. As ``++android:permission++`` is a uniform access control, if prevent client application form applying this principle.

Reviewer: ```suggestion
In regard to the Principle of Least Privilege, client applications that consume the content provider should only have the necessary privileges to complete their tasks. As ``++android:permission++`` is a uniform access control, it prevents client applications from applying this principle.
```

In regards to the "uniform access control": when I google for it I don't really have proper results for this term. Is it really a common one?

> In practice, it means that client application that requires read-only access to the content provider will always be granted both read and right permission. 

Reviewer: ```suggestion
In practice, it means client applications that require read-only access to the content provider will always be granted both read and write permissions. 
```
